### PR TITLE
Add missing tools for RNA workbench

### PR DESCRIPTION
**Pull request by bebatut**
ping @bgruening, @eggzilla, @jfallmann

### CODE REVIEW

File: rnateam.yml [hunk 1]
```diff
@@ -27,3 +27,47 @@ tools:
...
+  - name: fuma
+    owner: yhoogstrate
+    tool_panel_section_label: RNA Analysis 
+    
+  - name: infernal
+    owner: bgruening
+    tool_panel_section_label: RNA Analysis 
+    
+  - name: rnammer
+    owner: lionelguy
```
Comment (bgruening) [inline rnateam.yml]: is that true?

Comment (jfallmann) [reply rnateam.yml]: looks good, but there will be more to add, hopefully soon

Comment (bebatut) [reply rnateam.yml]: Same as in workbench: https://github.com/bgruening/galaxy-rna-workbench/blob/0de0d004bc3fc9a22761ed5713abc0c2a480ec81/rna_workbench_2.yml#L85

Comment (bgruening) [reply rnateam.yml]: It was updated last 2013, we should take it over or let it die ...

Comment (bgruening) [reply rnateam.yml]: Sorry, I'm more strict on EU than on our Docker.

Comment (jfallmann) [reply rnateam.yml]: is there a reason why we wouldn't take it over? Sorry maybe I missed the point

Comment (bgruening) [reply rnateam.yml]: No reason, just time. Make sure a conda-package is there and tests and get it into galaxytools.

Comment (bgruening) [reply rnateam.yml]: I removed it for now, please feel free to take it over to galaxytools.

**Comment by bgruening** [comment]
Installation is triggered manually. We will have all these tools soon on https://rna.usegalaxy.eu

**Pull request merged**